### ISSUE TITLE
Support zstd-fast compression options

### ISSUE THREAD

**Issue by danielkza** [openebs/zfs-localpv#501]
**Describe the problem/challenge you have**

zstd-fast compression option is not supported.

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

zfst-fast compression options are supported.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

```
E0207 17:50:59.680039       1 controller.go:957] error syncing claim "524024ec-2367-4052-b700-061db300370f": failed to provision volume with StorageClass "zfs-default": rpc error: code = Internal desc = not able to provision the volume, nodes [e11e9d9712994bccbb885e70e108e22c], err : ZFSVolume.zfs.openebs.io "pvc-524024ec-2367-4052-b700-061db300370f" is invalid: spec.compression: Invalid value: "zstd-fast": spec.compression in body should match '^(on|off|lzjb|zstd|zstd-[1-9]|zstd-1[0-9]|gzip|gzip-[1-9]|zle|lz4)$'
```

**Environment:**
- ZFS-LocalPV version
2.4.1 from Helm Chart

- Kubernetes version (use `kubectl version`):
1.29
- Kubernetes installer & version:
RKE2
- Cloud provider or hardware configuration:
Bare metal
- OS (e.g. from `/etc/os-release`):
Fedora 38.

**Comment by orville-wright** [comment]
thanks @danielkza 
Since the Zstandard lossless data compression algorithm has been included in Linux kernel from 2017 and OpenZFS since 2020, it feels reasonable that this option should be available in our ZFS-LocalPV Data Engine. 

I will track this feature request and see when we can schedule it for inserting into a future release.

**Comment by Abhinandan-Purkait** [comment]
@danielkza This has been added, you can try this with the latest develop tag. It should be available in 4.2 openebs release. Closing this.
